### PR TITLE
[DNS] Subdomain setups migration

### PR DESCRIPTION
**Pull request by kodster28**

### CODE REVIEW

File: content/dns/zone-setups/subdomain-setup/dnssec.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+pcx_content_type: how-to
+title: Enable DNSSEC
+weight: 4
+meta: 
+    title: Enable DNSSEC - Subdomain setup
+---
+
+# Enable DNSSEC for a Subdomain setup
```
Comment (marciocloudflare) [inline content/dns/zone-setups/subdomain-setup/dnssec.md]:
```suggestion
# Enable DNSSEC for a subdomain setup
```

File: content/dns/zone-setups/subdomain-setup/dnssec.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+pcx_content_type: how-to
+title: Enable DNSSEC
+weight: 4
+meta: 
+    title: Enable DNSSEC - Subdomain setup
```
Comment (marciocloudflare) [inline content/dns/zone-setups/subdomain-setup/dnssec.md]:
```suggestion
    title: Enable DNSSEC - subdomain setup
```

File: content/dns/zone-setups/subdomain-setup/dnssec.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+In order to use DNSSEC for a subdomain setup, DNSSEC must be enabled on the parent zone. 
+
+Ideally, you should also wait 12 to 24 hours after enabling DNSSEC on the parent zone to ensure DNS resolvers provide the same DNS query responses.
+
+## Setup
+
+1. [Create](/dns/zone-setups/subdomain-setup/setup/) the child subdomain.
+2. Make sure the child zone is [active](/dns/zone-setups/reference/domain-status/) on Cloudflare and that DNS resolution is working properly for your child subdomain.
+3. [Enable DNSSEC](/dns/additional-options/dnssec/) for the child subdomain and save the information provided within the `DS` Record output.
```
Comment (marciocloudflare) [inline content/dns/zone-setups/subdomain-setup/dnssec.md]:
```suggestion
3. [Enable DNSSEC](/dns/additional-options/dnssec/) for the child subdomain and save the information provided within the `DS` record output.
```

File: content/dns/zone-setups/subdomain-setup/dnssec.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+4. In the **DNS** settings of the parent domain, [add the `DS` Record](/dns/manage-dns-records/how-to/create-dns-records/) from the previous step.
```
Comment (marciocloudflare) [inline content/dns/zone-setups/subdomain-setup/dnssec.md]:
```suggestion
4. In the **DNS** settings of the parent domain, [add the `DS` record](/dns/manage-dns-records/how-to/create-dns-records/) from the previous step.
```

File: content/dns/zone-setups/subdomain-setup/dnssec.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+    ![Screenshot showing how to add a DS record within Cloudflare](/dns/static/ds-record-example.png)
+
+5. Add an `A` record to the child subdomain to validate DNS resolution.
+6. Wait two to six hours. Then, [test the `A` record](https://support.cloudflare.com/hc/en-us/articles/360021111972#TroubleshootingDNSSEC-DNSSECinPracticewithDig) added in the previous step using multiple DNS resolvers with DNSSEC validation (`1.1.1.1`, `8.8.8.8`, and `9.9.9.9`). For example, if the A record is for `test.child.example.com`: `dig test.child.example.com +dnssec @1.1.1.1`.
```
Comment (marciocloudflare) [inline content/dns/zone-setups/subdomain-setup/dnssec.md]:
```suggestion
6. Wait two to six hours. Then, [test the `A` record](https://support.cloudflare.com/hc/en-us/articles/360021111972#TroubleshootingDNSSEC-DNSSECinPracticewithDig) added in the previous step using multiple DNS resolvers with DNSSEC validation (`1.1.1.1`, `8.8.8.8`, and `9.9.9.9`). For example, if the `A` record is for `test.child.example.com`: `dig test.child.example.com +dnssec @1.1.1.1`.
```

File: content/dns/zone-setups/subdomain-setup/setup.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
+---
+pcx_content_type: how-to
+title: Setup
```
Comment (marciocloudflare) [inline content/dns/zone-setups/subdomain-setup/setup.md:3]: This should probably also be `Set up` but that's weird to see on a side menu, right?

File: content/dns/zone-setups/subdomain-setup/setup.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+### Subdomain already exists in the parent domain
+
+If you have already created a DNS record covering your child domain in the parent domain:
+
+1. [Add the child domain](/fundamentals/get-started/setup/add-site/) to the parent domain's Cloudflare account or another account.
+2. In your child domain, [re-create all DNS records](/dns/manage-dns-records/how-to/create-dns-records/) that relate to your child domain. This includes all DNS records deeper than the delegated subdomain, meaning that if you are delegating `www.example.com`, you should also move over records for `api.www.example.com`.
+
+    {{<Aside type="note">}}Cloudflare recommends [exporting](/dns/manage-dns-records/how-to/import-and-export/#export-records) records from the parent domain, deleting all unnecessary records, and then [importing](/dns/manage-dns-records/how-to/import-and-export/#import-records) the records into your new zone.
+    {{</Aside>}}
+    <br/>
```
Comment (marciocloudflare) [inline content/dns/zone-setups/subdomain-setup/setup.md]: Any reason for the break thingy?

File: content/dns/zone-setups/subdomain-setup/setup.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+
+If you have already created a DNS record covering your child domain in the parent domain:
+
+1. [Add the child domain](/fundamentals/get-started/setup/add-site/) in the same or a new account.
+2. Convert the child zone to [a partial setup](/dns/zone-setups/partial-setup/setup/#step-1--add-your-domain-to-cloudflare).
+3. In your child domain, [re-create all DNS records](/dns/manage-dns-records/how-to/create-dns-records/) that relate to your child domain. This includes all DNS records deeper than the delegated subdomain, meaning that if you are delegating `www.example.com`, you should also move over records for `api.www.example.com`.
+
+    {{<Aside type="note">}}Cloudflare recommends [exporting](/dns/manage-dns-records/how-to/import-and-export/#export-records) records from the parent domain, deleting all unnecessary records, and then [importing](/dns/manage-dns-records/how-to/import-and-export/#import-records) the records into your new zone.
+    {{</Aside>}}
+    <br/>
```
Comment (marciocloudflare) [inline content/dns/zone-setups/subdomain-setup/setup.md]: Same here

**Review by marciocloudflare — APPROVED** [review]

File: content/dns/zone-setups/subdomain-setup/setup.md (hunk 6 above)
Comment (kodster28) [reply content/dns/zone-setups/subdomain-setup/setup.md:3]: Yeah, that's what I was thinking

**Pull request merged**